src/QMCWaveFunctions/tests/SetupDiracDeterminantResources.hpp

```diff
@@ -0,0 +1,53 @@
+//////////////////////////////////////////////////////////////////////////////////////
```

Comment:
I think this will be dropped as it was supposed to provide generic testing but made bad resource management harder to find.